### PR TITLE
feat(wasm) prepare package for alpha publish

### PR DESCRIPTION
**Pull request by lucasfernog**

### CODE REVIEW

File: bindings/wasm/package.json [hunk 1]
```diff
@@ -14,8 +17,14 @@
     "prepublishOnly": "yarn build",
     "test": "mocha"
   },
-  "author": "Lucas Nogueira <lucas.nogueira@iota.org>",
-  "license": "ISC",
+  "contributors": [
+    "Lucas Nogueira <lucas.nogueira@iota.org>"
+  ],
+  "license": "Apache-2.0",
```
Comment (wusyong) [inline bindings/wasm/package.json]: Why is it not MIT here?

**Review by wusyong — COMMENTED** [review]
We probably should mention license in Cargo.toml too.

File: bindings/wasm/package.json (hunk 1 above)
Comment (nothingismagick) [reply bindings/wasm/package.json]: My understanding is that Apache-2.0 is the license for all IOTA projects moving forward, but yes, we should use also use MIT here for historical reasons... (original license is MIT) 

so: MIT or Apache 2 (just like the rust ecosystem) ?

Comment (wusyong) [reply bindings/wasm/package.json]: yeah, I think it would be great to be MIT or Apache2 like what we usually do on general crates.

**Pull request merged**